airflow/providers/amazon/aws/hooks/base_aws.py

```diff
@@ -393,7 +394,7 @@ def get_session(self, region_name: Optional[str] = None) -> boto3.session.Sessio
         session, _ = self._get_credentials(region_name)
         return session
 
-    def get_credentials(self, region_name: Optional[str] = None) -> Tuple[Optional[str], Optional[str]]:
```

Comment:
The `botocore.credentials.ReadOnlyCredentials` is namedtuple. It's not tuple.  To avoid typing error in `s3_to_redshift.py`  this modification is necessary.